README.md

```diff
@@ -1,5 +1,5 @@
-pkg - a binary package manager for FreeBSD
-============================================
+pkg - a binary package manager for FreeBSD and OpenBSD
+========================================================
```

Comment:
Hmmm, saying pkg is for FreeBSD makes its seem like it won't run on any other OS. Maybe there's a better way of explaining this point.